app/helpers/application_helper.rb

```diff
@@ -1,2 +1,7 @@
 module ApplicationHelper
+  def signup_link
+    unless current_page?(signup_path)
+      content_tag(:li, (link_to 'Sign Up', signup_path))
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.